### DISCUSSION TITLE
extension

### DISCUSSION THREAD

**Discussion by Ali-Key** [sharafdin/markdown-master#8] [Q&A] [answered]
maxaa iga caawinaaayo inan extension aan sameysto

**Comment by miirshe** [comment, marked as answer]
waxaa kaa caawinkaro waa inaad research aad sameyo sida microsoft vscode documentation oo kale page kooda tag waxaa jiro wax recomendations ah iyo requirement ah marka aad sameyneyso tusaale waxaad rabtaa inaad extension ka aad sameyneyso shortcuts
ad u sameysid waxii shortcuts ah package.json gaar ahaan keywords ayaad galineysaa iwm
